packages/@glimmer/debug-util/lib/platform-utils.ts

```diff
@@ -16,7 +16,7 @@ export function unwrap<T>(val: Maybe<T>): T {
  * existing `LOCAL_DEBUG` check.
  */
 export function dev<T>(val: Optional<() => T>): T {
-  if (val === null || val === undefined) {
```

Comment:
is it no longer possible for val to be null?

Reply:
It was never possible based on the type, and this code is only used internally by other typed code.